components/MemberComponent.vue

```diff
@@ -1,9 +1,13 @@
 <template>
     <div class="member flex flex-col items-center text-center">
-        <img
-            data-testid="member-avatar"
-            :src="avatarImg"
-            class="member-image object-cover rounded-full w-32 h-32"
+        <SmoothLoadingImage
```

Comment:
These props allow you to customize the image to the place you want to use this component since we might have more assets in the future